bin/millau/runtime/src/rialto_messages.rs

```diff
@@ -87,9 +88,15 @@ impl MessageBridge for WithRialtoMessageBridge {
 	type ThisChain = Millau;
 	type BridgedChain = Rialto;
 
-	fn maximal_dispatch_weight_of_message_on_bridged_chain() -> Weight {
+	fn weight_limits_of_message_on_bridged_chain(message_payload: &[u8]) -> RangeInclusive<Weight> {
 		// we don't want to relay too large messages + keep reserve for future upgrades
-		bp_rialto::MAXIMUM_EXTRINSIC_WEIGHT / 2
+		let upper_limit = bp_rialto::MAXIMUM_EXTRINSIC_WEIGHT / 2;
+
+		// given Rialto chain parameters (`TransactionByteFee`, `WeightToFee`, `FeeMultiplierUpdate`),
+		// the minimal weight of the message may be computed as message.length()
+		let lower_limit = Weight::try_from(message_payload.len()).unwrap_or(Weight::MAX);
```

Comment:
So by having the "default" be `Weight::MAX` we basically invalidate any range returned afterwards since nothing can be contained in the Range `Weight::MAX..=upper_limit`, right?

Reply:
Right